committing.rst

```diff
@@ -250,6 +250,23 @@ understands the justification for the change).  Also, if a non-core developer
 contributed to the resolution, it is good practice to credit them.
 
 
+Reverting a Commit
+------------------
+
+To revert a merged pull request, press the ``Revert`` button at the bottom of
+the pull request.  It will bring up the page to create a new pull request where
+the commit can be reverted.
```

Comment:
Let's also mention that just clicking the revert button is enough to create a branch on the main repository that should be deleted when the reversion PR is merged.